assets/css/_view_header.scss

```diff
@@ -1,5 +1,7 @@
 .m-view-header {
-  position: relative;
+  position: sticky;
```

Comment:
I saw that some other uses of `position: sticky` in this repo also specified `position: -webkit-sticky`. I don't see that prefix specified in the [mdn doc](https://developer.mozilla.org/en-US/docs/Web/CSS/position#browser_compatibility), so I'm thinking maybe it is an artifact of an old version and am not including. If it should be included though I'll add it!

Reply:
[Per Can I Use](https://caniuse.com/?search=sticky) it looks like `-webkit-sticky` was needed for certain versions of Safari but those are quite old at this point. We do actually have a [browser support policy](https://www.notion.so/mbta-downtown-crossing/Skate-Browser-Support-Policy-7cf57f38ea294afb889e63e95f73a300) that we put together a while back and we require a much newer version of Safari, so I'd say you're good to leave out `-webkit-sticky` or remove it if you're ever modifying code.